src/non_windows.rs

```diff
@@ -1,3 +1,27 @@
-pub const SUPPORTED: bool = false;
+use std::env;
+use std::process::Command;
 
-pub fn compile_resource(_: &str, _: &str, _: &str) {}
+pub static SUPPORTED: bool = true;
```

Comment:
No it's bloody not; if anything. `windres` variations could go into `windows_not_msvc`.